tools/abel/validators/README.md

```diff
@@ -0,0 +1,192 @@
+only supports Green Field buildings or Building Configs under the [Initialize
+operation](../../../ontology/docs/building_config.md#initialize)
+
+#### Green Field
+
+This is the case where ABEL is used to create a Building Config from scratch. In
+this case **GUIDs do not need to be provided except for when there are
+duplicate entity codes.** Then and only then will the user need to provide
+GUIDs for the entities with duplicate codes in every one of the ABEL
+spreadsheet tables.
+
+***Note:*** ABEL can ingest Building Config files using the [initialize
+operation](../../../ontology/docs/building_config.md#initialize) but it does not
+currently support [update
+operations](../../../ontology/docs/building_config.md#update)
+
+### Site
+
+The `Site` tab holds all information about a project site or building.
+
+`Building Code` *string* **required**
+
+A [human-readable code](../../../ontology/docs/building_config.md#identifiers) for a building. e.g. `US-SVL-TC2`
+
+`Entity Guid` *string*
+
+[UUID4](https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_(random)) [identifier](../../../ontology/docs/building_config.md#identifiers) for a building.
+Unless you are onboarding a Google building, this value is generated by ABEL and does not need to be provided for initial onboarding.
+
+***Note:*** A Site should not be considered an entity and should not be listed in
+the `Entities` tab.
+
+### Entities
+
+The `Entities` tab holds all information about [reporting entities](../../../ontology/docs/building_config.md#reporting-physical-devices) and [virtual entities](../../../ontology/docs/building_config.md#virtual-devices) in a building or site. If ABEL is being used for Google then Guids are required for all entities in the `FACILTITIES` namespace which can be retrieved from DB API.
+
+`Entity Code` *string* **required**
+
+[Human-readable code](../../../ontology/docs/building_config.md#identifiers) for an entity e.g. `AHU-1001`
+
+`Entity Guid` *string*
+
+[UUID4](https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_(random))
+[identifier](../../../ontology/docs/building_config.md#identifiers) for an entity. **Note:** If an entity belongs to the `FACITITLIES` namespace and ABEL is
+being used or Google, then this field is **required** or else spreadsheet
+validation will fail.
+
+`Etag` *string*
+
+Version control tag for entities in an entity type database. Used only when updating an existing building config.
+
+`Is Reporting` *Boolean* **required**
+
+`TRUE` if an entity is [reporting its own data](../../../ontology/docs/building_config.md#reporting-physical-devices) and `FALSE` if [the entity is virtual](../../../ontology/docs/building_config.md#virtual-devices).
+
+`Cloud Device ID` *string*
+
+Cloud registry [device numeric identifier](../../../ontology/docs/building_config.md#identifiers). The device numeric ID is automatically created by a Cloud IoT application, and it is globally unique and not editable.
```

Comment:
it not really an application. you can mention `Cloud IoT Core` instead of application